### DISCUSSION TITLE
Oversight - Mac 相機和麥克風開啟通知

### DISCUSSION THREAD

**Discussion by gnsJhenJie** [1101-GS4719-C/Python-Community-GS4719-C#633] [優秀專案]
https://github.com/objective-see/OverSight
這個專案的程式會在mac的麥克風或相機被開啟時傳送通知，除了可以知道應用程式是否偷開麥克風或相機外，也可以在自己誤開十多一個提醒。
程式傳送通知的樣子：
<img width="370" alt="Screen Shot 2022-05-25 at 14 08 09" src="https://user-images.githubusercontent.com/32983347/170192768-ad04f877-2a7b-441c-ae8d-d8a415252a53.png">

**Comment by leon10456** [comment]
感謝樓主的分享

**Comment by liangyinglly** [comment]
上網查了更多資料後發現
雖然mac鏡頭啟用時會亮綠燈但惡意程式可能透過一些合法啟動鏡頭後再進行不良運用
因此全憑綠燈來判斷是否被監視不太安全 仍需要這種程式的幫忙才能完全防堵被監視

**Comment by jcpan821** [comment]
這樣就能減少誤開的時間，不然上課都會被干擾
